resources/views/inputs/tile-selection.blade.php

```diff
@@ -72,6 +83,8 @@ class="form-checkbox tile-selection-select-all-checkbox"
                         'option' => $option,
                         'wireModel' => $single ? ($model ?? $id) : ($model ?? $id).'.'.$option['id'].'.checked',
                         'mobileHidden' => $loop->index >= ($mobileShowRows * 2),
+                        'isDisabled' => $withoutSelectAll ? ($selectedOptionsCount >= $selectionLimit ? true : false) : false,
```

Comment:
why explicit `true` and `false` values instead of relying on the responses that are already booleans?